packages/agoric-cli/src/commands/gov.js

```diff
@@ -63,11 +83,15 @@ export const makeEconomicCommiteeCommand = (_logger, io = {}) => {
    * @param {{
    *   toOffer: (agoricNames: *, current: import('@agoric/smart-wallet/src/smartWallet').CurrentWalletRecord | undefined) => OfferSpec,
    *   sendFrom?: string | undefined,
+   *   keyringBackend: string,
    *   instanceName?: string,
    * }} detail
    * @param {Awaited<ReturnType<makeRpcUtils>>} [optUtils]
    */
-  const processOffer = async function ({ toOffer, sendFrom }, optUtils) {
+  const processOffer = async function (
```

Comment:
can these use [arrow function style](https://github.com/Agoric/agoric-sdk/wiki/Arrow-Function-Style) as well?

Reply:
i believe Commander.js relies upon the `this` binding.